app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
```

Comment:
- [ ] The same here goes for the reference to the `Author` with class_name to the user.